supervisor/backups/backup.py

```diff
@@ -485,6 +474,26 @@ def restore_homeassistant(self):
         # save
         self.sys_homeassistant.save_data()
 
+    async def store_homeassistant_config_dir(self):
+        """Backup Home Assitant Core configuration folder."""
+
+        # Backup Home Assistant Core config directory
+        homeassistant_file = SecureTarFile(
+            Path(self._tmp.name, "homeassistant.tar.gz"), "w", key=self._key
+        )
```

Comment:
why not just handle that all on sys_homeassistant.backup?

Reply:
This aligns it with how add-ons work: The `SecureTarFile` is also handled in backup.py.

I think its good the way it is: The `Backup` class should know the details about the tar file (where its stored/name, etc.)